starwars/src/components/Film.jsx

```diff
@@ -0,0 +1,42 @@
+import React,{Component} from 'react'
+
+export default class Film extends Component {
+    constructor(props){
+        super(props)
+        this.state={
+            film:null
+        }
+    }
+
+    componentDidMount(){
+        this.getFilm(this.props.filmURL)
+    }

```

Comment:
Great job setting another fetch, and pulling down the film data nested in another url in the SWAPI data originally received by the app.